packages/back-end/src/routers/demo-datasource-project/demo-datasource-project.controller.ts

```diff
@@ -0,0 +1,404 @@
+  database: "sample",
+  password: "",
+  port: 5432,
+  ssl: false,
+  defaultSchema: "",
+};
+
+const ASSET_OWNER = "datascience@growthbook.io";
+const DEMO_TAGS = ["growthbook-demo"];
+
+// Metric constants
+const DEMO_METRICS: Pick<
+  MetricInterface,
+  | "name"
+  | "description"
+  | "type"
+  | "sql"
+  | "conversionWindowHours"
+  | "conversionDelayHours"
+  | "aggregation"
+>[] = [
+  {
+    name: "Purchases - Total Revenue (72 hour window)",
+    description: "The total amount of USD spent aggregated at the user level",
+    type: "revenue",
+    sql:
+      "SELECT\nuserId AS user_id,\ntimestamp AS timestamp,\namount AS value\nFROM orders",
+  },
+  {
+    name: "Purchases - Any Order (72 hour window)",
+    description: "Whether the user places any order or not (0/1)",
+    type: "binomial",
+    sql: "SELECT\nuserId AS user_id,\ntimestamp AS timestamp\nFROM orders",
+  },
+  {
+    name: "Purchases - Number of Orders (72 hour window)",
+    description: "Total number of discrete orders placed by a user",
+    type: "count",
+    sql:
+      "SELECT\nuserId AS user_id,\ntimestamp AS timestamp,\n1 AS value\nFROM orders",
+  },
+  {
+    name: "Retention - [1, 14) Days",
+    description:
+      "Whether the user logged in 1-14 days after experiment exposure",
+    type: "binomial",
+    conversionDelayHours: 24,
+    conversionWindowHours: 144 + 168,
+    sql:
+      "SELECT\nuserId AS user_id,\ntimestamp AS timestamp\nFROM pages WHERE path = '/'",
+  },
+  {
+    name: "Days Active in Next 7 Days",
+    description:
+      "Count of times the user was active in the next 7 days after exposure",
+    type: "count",
+    conversionWindowHours: 168,
+    aggregation: "COUNT(DISTINCT value)",
+    sql:
+      "SELECT\nuserId AS user_id,\ntimestamp AS timestamp,\nDATE_TRUNC('day', timestamp) AS value\nFROM pages WHERE path = '/'",
+  },
+];
+
+const DEMO_RATIO_METRIC: Pick<
+  MetricInterface,
+  "name" | "description" | "type" | "sql"
+> = {
+  name: "Purchases - Average Order Value (ratio)",
+  description:
+    "The average value of purchases made in the 72 hours after exposure divided by the total number of purchases",
+  type: "revenue",
+  sql:
+    "SELECT\nuserId AS user_id,\ntimestamp AS timestamp,\namount AS value\nFROM orders",
+};
+
+// Feature constants
+const DEMO_FEATURES: Omit<
+  FeatureInterface,
+  "id" | "organization" | "dateCreated" | "dateUpdated"
+>[] = [
+  {
+    description:
+      "Controls checkout layout UI. Employees forced to see new UI, other users randomly assigned to one of three designs.",
+    owner: ASSET_OWNER,
+    valueType: "string",
+    defaultValue: "current",
+    tags: DEMO_TAGS,
+    environmentSettings: {
+      production: {
+        enabled: true,
+        rules: [
+          {
+            type: "force",
+            description: "",
+            id: "gbdemo-checkout-layout-employee-force-rule",
+            value: "dev",
+            condition: "is_employee = true",
```

Comment:
Conditions must be a JSON formatted string.

```suggestion
            condition: `{"is_employee":true}`,
```